vtkext/private/module/vtkF3DEXRReader.cxx

```diff
@@ -38,13 +88,13 @@ void vtkF3DEXRReader::ExecuteInformation()
   this->ComputeInternalFileName(this->DataExtent[4]);
   if (this->InternalFileName == nullptr || this->InternalFileName[0] == '\0')
   {
-    return;
+    // If we have no file then maybe we have the file in memory
+    if (!this->MemoryBuffer)
+      return;
```

Comment:
{}

Reply:
should this be a "return {}" ?
I do not see that being used anywhere else in the file. should i change them all to be like that?

Reply:
Sorry, I should have been clearer.

You are missing {}, it should look like:

```
    if (!this->MemoryBuffer)
    {
      return;
    }
```